### DISCUSSION TITLE
微信v3.7.0.29 版本有如图报错

### DISCUSSION THREAD

**Discussion by lxl217114** [huiyadanli/RevokeMsgPatcher#356] [Q&A] [answered]
微信v3.7.0.29 版本有如图报错，麻烦作者看看是啥问题。
如下图：
![BQV JK~JCR%I}{TORF {JR2](https://user-images.githubusercontent.com/54263659/171411728-7c1bda6e-9e14-432f-86c1-2835773f92e4.png)

**Comment by woxihuanchizhangzhongbao** [comment]
同，也显示这个提示，但是QQ是正常的

**Comment by huiyadanli** [comment, marked as answer]
不勾选去除校验即可

参考 #353 #355 

或者直接下载最新编译版本就不存在这个问题了 https://github.com/huiyadanli/RevokeMsgPatcher/actions/runs/2448013916

**Comment by woxihuanchizhangzhongbao** [comment]
成功了，牛逼，谢谢。



















在 2022-06-06 22:30:15，"辉鸭蛋" ***@***.***> 写道：

不勾选去除校验即可

参考 #353#355

或者直接下载最新编译版本就不存在这个问题了 https://github.com/huiyadanli/RevokeMsgPatcher/actions/runs/2448013916

**Comment by fuxin-mail** [comment]
最新版微信3.7.0.30有崩溃[https://github.com/fuxin-mail/pic/blob/6d19495a82765bcfc73e47ae7412ec8a7e97f15b/wx.png](url)
